src/connector/postgres.rs

```diff
@@ -85,12 +85,12 @@ impl SslParams {
         auth.accept_mode(self.ssl_accept_mode);
 
         if let Some(ref cert_file) = self.certificate_file {
-            let cert = fs::read(cert_file).await?;
+            let cert = fs::read(cert_file)?;
```

Comment:
hmmm I don't know what the right thing to do would be here if we don't want to always depend on async-std, but reverting to blocking file IO feels bad

Reply:
I don't want to rely on a runtime that spawns random threads on connect. The facilities should come from the current runtime or we just block. More we rely on certain runtime functions, harder it is to support other runtimes.